src/content/docs/fr/guides/routing.mdx

```diff
@@ -407,21 +407,27 @@ interface Page<T = any> {
 	total: number;
 	/** Numéro de la page actuelle, à partir de 1 */
 	currentPage: number;
-	/** Nombre d'éléments par page (par défaut : 25) */
+	/** Nombre d'éléments par page (par défaut : 10) */
 	size: number;
 	/** Nombre de la dernière page */
 	lastPage: number;
 	url: {
 		/** URL de la page actuelle */
 		current: string;
-		/** URL de la page précédente (si il y en a une) */
+		/** URL de la page précédente (s'il y en a une) */
 		prev: string | undefined;
-		/** URL de la page suivante (si il y en a une) */
+		/** URL de la page suivante (s'il y en a une) */
 		next: string | undefined;
+		/** url de la première page (si la page actuelle n'est pas la première page) */
+		first: string | undefined;
+		/** url de la page suivante (si la page actuelle n'est pas la dernière) */
+		last: string | undefined;
 	};
 }
 ```
 
+<ReadMore>En savoir plus sur [la prop `page` de pagination](/fr/reference/api-reference/#la-prop-page-de-pagination).</ReadMore>
```

Comment:
Since you choose `la propriété` instead of `la prop` in #8922 (see https://deploy-preview-8922--astro-docs-2.netlify.app/fr/reference/api-reference/#la-propri%C3%A9t%C3%A9-page-de-pagination) we need this update:
```suggestion
<ReadMore>En savoir plus sur [la prop `page` de pagination](/fr/reference/api-reference/#la-propriété-page-de-pagination).</ReadMore>
```